documentation/api/command/v1/commands.markdown

```diff
@@ -204,6 +208,16 @@ The payload is expected to be a report, containing events that occurred on
 Puppet resources. It is structured as a JSON object, conforming to the
 [report wire format v5][reportv5].
 
+### "configure expiration", version 1 (experimental)
+
+The payload should be a JSON format command, conforming to the
+[configure_expiration wire format v1][expirev1], indicating whether or
```

Comment:
I think you mean `[]()` here, not `[][]`

Reply:
Hmm, I think [][] may be right for a "reference link" (it's pointing to the top of the file) -- see the other examples above and below.